duckduckgo/iOS
# support future nav property in autocomplete results

Task/Issue URL: https://app.asana.com/0/230839424936/1200348293580045
Tech Design URL:
CC:  @gtfiorentino

**Description**:

Changes the parsing of the autocomplete results to not break if unexpected properties are presented and also supports upcoming nav property.

**Steps to test this PR**:

1. Type something in to the address bar.  Only suggestions with isNav: true should now be navigational.  `healthcare.gov` is a good search.

Note: assumption is that this will be released on the server later today.  If not, update the in app end point to point at a dev instance (see task above).

***General***

1. Check that searching for bookmarks via the address bar still works as expected.



**Device Testing**:

**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
